### ISSUE TITLE
Instantiate a ScatModel object by loading in a FITS table

### ISSUE THREAD

**Issue by eblur** [eblur/newdust#11]
Use the output of `ScatModel.write_table` to instantiate a custom `ScatModel` object.

**Comment by eblur** [comment]
Started working on this in branch `read_tables`

Can read in using `ScatModel(from_file='filename.fits')`

However, to calculate everything in a SingleGrainPop, you currently need an extra call of `SingleGrainPop.calculate_ext`

Make it streamlined so a scattering table model can be implemented in one call, e.g.
```
from newdust import SingleGrainPop
my_pop = SingleGrainPop('Powerlaw', 'Silicate', 'CustomName', scatm_from_file='filename.fits')
```

**Comment by eblur** [comment]
Implemented in PR #14
